build/tools/builder.sh

```diff
@@ -0,0 +1,235 @@
+#!/bin/bash
+
+# This script is supposed to automate various types of artefact generation that this project has
+# to do.
+# Following types of artefacts are supported:
+# 1. Build a new version of the CLI
+# 2. Build a new version of OCI image
+# 3. Generate Krew manifest
+# 4. Generate Homebrew formula
+
```

Comment:
We might want to add PS4 for this script to see the debugging:
```suggestion

export PS4='\e[36m+ ${FUNCNAME:-main}@${BASH_SOURCE}:${LINENO} \e[0m'
```